syntax = "proto3";
package duchess.v1alpha1;

import "duchess/v1alpha1/positions.proto";


message Game {
    string id = 1;
    duchess.v1alpha1.Position position = 2;
}

message GetSimilarGamesRequest {
    string position_fen = 1;
}
message GetSimilarGamesResponse {
    repeated duchess.v1alpha1.Game games = 1;
}
